input/postgres/explain.go

```diff
@@ -77,3 +95,12 @@ func RunExplain(db *sql.DB, connectedDbName string, inputs []state.PostgresQuery
 
 	return
 }
+
+func contains(strs []string, val string) bool {
```

Comment:
Is there a built-in way to do this that I missed? I expected `strings` maybe to have something but I didn't see anything.